app/src/main/java/cm/aptoide/pt/install/InstallManager.java

```diff
@@ -226,11 +230,14 @@ private Completable install(Download download, boolean forceDefaultInstall,
         .map(storedDownload -> updateDownloadAction(download, storedDownload))
         .retryWhen(errors -> createDownloadAndRetry(errors, download))
         .doOnNext(storedDownload -> {
+          aptoideInstallManager.addAptoideInstallCandidate(storedDownload.getPackageName());
           if (storedDownload.getOverallDownloadStatus() == Download.ERROR) {
             storedDownload.setOverallDownloadStatus(Download.INVALID_STATUS);
             downloadRepository.save(storedDownload);
           }
         })
+        .flatMap(install -> aptoideInstallManager.sendConversionEvent()
```

Comment:
should the conversion event be sent when we start the download or just when we know it was installed? It is kind of irrelevant but just putting it out there

Reply:
Well it is not clear on the ticket when should it convert so I have no idea. I assumed it was on click, but might want to check with Ana